pcre2-sys/build.rs

```diff
@@ -93,9 +93,13 @@ fn main() {
         .define("PARENS_NEST_LIMIT", "250")
         .define("PCRE2_STATIC", "1")
         .define("STDC_HEADERS", "1")
-        .define("SUPPORT_JIT", "1")
         .define("SUPPORT_PCRE2_8", "1")
         .define("SUPPORT_UNICODE", "1");
+    if !target.starts_with("aarch64-apple") &&
```

Comment:
Could you add a comment before this `if` block explaining why we don't enable the JIT in these cases?

I might prefer defining a function, `fn enable_jit(builder: &mut cc::Build)` instead, and documenting that as appropriate.